lib/index.js

```diff
@@ -276,17 +276,25 @@ module.exports = class SVGSpritemapPlugin {
         // Clean up the spritemap chunk
         compiler.hooks.emit.tapAsync(plugin, (compilation, callback) => {
             const { output: outputOptions } = this.options;
+            const assets = compilation.assets;
+            
 
             if ( outputOptions.chunk.keep ) {
                 return callback();
             }
 
             compilation.chunks.forEach((chunk) => {
+                const sourceMapStr = `${chunk.files[0]}.map`;
```

Comment:
Is the sourcemap `.map` file not included in `chunk.files`? 🤔

Reply:
The sourcemap files are in `chunk.files`. Sorry! I'm not sure I understand the question. Can you clarify?

Reply:
I think I got you mean. The sourcemap file, if one exists, is at the 2nd index (`chunk.files[1]`) I did the string template so we could ensure we're always just cleaning up the sourcemap file for the pertinent chunk. I could do a check for the `.map` extension on the 2nd index and delete it if it's true. Would that be clearer?

Reply:
We can just do the following on line 294 (I think, might need some testing) as this will only delete the index when the chunk name matches (which is checked on line 289).

```es6
if ( typeof chunk.files[1] !== 'undefined' ) {
    delete assets[chunk.files[1]];
}
```

Reply:
Sounds good. I'll test that now.

Reply:
Tested and pushed.